examples/sprite_animation/assets/anims/bat.ron

```diff
@@ -0,0 +1,13 @@
+{
+    "Sampler": (
+                                            
+                                            // Time of key frames
+                                            input: [0.0, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.1, 1.2, 1.3, 1.4],
+                                            // Sprite indexes from SpriteSheet for key frames
+                                            output: [5, 4, 3, 2, 1, 0, 1, 2, 3, 4, 4],
+                                            // Sprites can only ever be animated with Step
+                                        function: Step,
+                                        
+
+)
+}
```

Comment:
newline

Reply:
we need a ron formatter, but the existing one doesn't support comments.  I will see if I can make a PR to their project